scheds/rust/scx_cosmos/src/bpf/main.bpf.c

```diff
@@ -325,13 +325,13 @@ static inline const struct cpumask *get_idle_smtmask(s32 cpu)
  */
 static inline bool is_cpu_idle(s32 cpu)
 {
-	struct rq *rq = scx_bpf_cpu_rq(cpu);
+	struct task_struct *p = __COMPAT_scx_bpf_cpu_curr(cpu);
```

Comment:
We need to add the `__COMPAT_scx_bpf_cpu_curr()` https://git.kernel.org/pub/scm/linux/kernel/git/tj/sched_ext.git/tree/tools/sched_ext/include/scx/compat.bpf.h?h=for-6.18#n238 to `scheds/include/scx/compat.bpf.h` or this won't build.